sparrow-py/pysrc/sparrow_py/_timestream.py

```diff
@@ -485,10 +489,14 @@ def ne(self, other: Union[Timestream, Literal]) -> Timestream:
         -------
         Timestream
             The Timestream indicating whether `self` and `other` are not equal.
+
+        Note
+        ----
+        Inequality is *not* available as `a != b`.
         """
         return Timestream._call("neq", self, other)
 
-    def __getitem__(self, key: Union[Timestream, Literal]) -> Timestream:
+    def index(self, key: Union[Timestream, Literal]) -> Timestream:
```

Comment:
So when we have maps we'll do `map.index("field_name")`?

Reply:
Or, `map["field_name"]` I suppose is available too?

Reply:
We can do either -- `a[b]` calls `a.index[b]`. It seemed cleaner to say "there is a function called index, and it does X, Y, Z, and you can write it as `a[b]` for short" than to document `__getitem__` directly.